health_vara.py

```diff
@@ -73,22 +74,20 @@ def __setup__(cls):
 
 
 screening_types = [
-    (None, ''),
     ('screening_first_time',
         'Screening: First-time Routine Screening Visit'),
     ('screening_return', 'Screening: Routine Annual Screening'),
     ('screening_second_opinion', 'Screening: Get Second Opinion'),
     ]
 diagnostic_types = [
-    (None, ''),
     ('diagnostic_recall', 'Diagnostic: Recall'),
     ('diagnostic_short_term',
         'Diagnostic: Short-term follow-up '
         '(~6 months since last screening visit)'),
     ('diagnostic_symptoms', 'Diagnostic: Breast symptoms'),
     ('diagnostic_post_treat', 'Diagnostic: Post-treatment surveillance'),
     ]
-vara_types = screening_types + diagnostic_types
+vara_types = [(None, '')] + screening_types + diagnostic_types
```

Comment:
Why is this `None` type necessary now with Arabic translations added?

Reply:
we want a nil option in the list of options, and this says that the nil option should show as an empty string.
Before, when we combined the two lists, we ended up with two nil options so this cleans that up.